modules/acs-quick-install-secured-cluster-using-helm.adoc

```diff
@@ -14,10 +22,15 @@ To install Collector on systems that have Unified Extensible Firmware Interface
 ====
 
 .Prerequisites
+ifndef::cloud-svc[]
 * You must have the address and the port number that you are exposing the Central service on.
+endif::cloud-svc[]
+ifdef::cloud-svc[]
+* You must have the *Central API Endpoint*, or the URI, including the address and the port number, of the Central service. You can view this information by choosing *Advanced Cluster Security* -> *ACS Instances* from the cloud console navigation menu, then clicking the ACS instance you created.
```

Comment:
_the *Central API Endpoint*, or the URI_ As I understand the grammar, this means that the Central API Endpoint and the URI are the same thing. Are they the same thing?

Reply:
I think it used to say "URI" somewhere either here or in the secured cluster, but now it doesn't, so I am going to remove. It just uses "Central API endpoint" now.